tarantool/__init__.py

```diff
@@ -40,7 +40,11 @@
     Interval,
 )
 
-__version__ = "0.9.0"
+try:
+    from tarantool.version import VERSION
+    __version__ = VERSION
+except ImportError:
+    __version__ = 'dev'
```

Comment:
It is an idiomatic way, see https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package
```suggestion
try:
    from tarantool._version import __version__
except ImportError:
    __version__ = 'dev'
```

Reply:
Thanks, updated